app/views/works/_meta.html.erb

```diff
@@ -73,6 +73,16 @@
              <%= @work.ip_address %>
           <% end %>
         </dd>
+        <dt class="akismetspam">
```

Comment:
Actually, ignore my previous comment! `akismet` is a better class name (we can save `spam` for any sort of homegrown spam-slaying items). Also, We capitalize the `<dt>` labels in the work meta, so it should be `Akismet Reported as Spam:` on line 77.